File: How Many Numbers Are Smaller Than the Current Number.js
var smallerNumbersThanCurrent = function(nums) {
    let arr = [];
    nums.map(n => {
        let i = 0;
        nums.forEach(nn => {
            if(n>nn) i++;
        });
        
        arr.push(i);
    });
    return arr;
};

File: A2SV_programming/how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        res = []
        for i in range(len(nums)):
            count = 0
            for j in range(len(nums)):
                if nums[i] != nums[j] and nums[i] > nums[j]:
                    count += 1
            res.append(count)
        return res
        